docs/compilers/CSharp/Warnversion Warning Waves.md

```diff
@@ -0,0 +1,25 @@
+﻿# /warnversion warning "waves"
```

Comment:
Will this flag be added/respected for VB as well?

Reply:
my understanding is that it would not be at this time, but it could be added later if there are warnings we want to add